src/state/upload/upload-reducers.js

```diff
@@ -37,9 +38,24 @@ export default function (state = initialState, action) {
 
         case uploadActionTypes.FETCH_ITEMS_SUCCEEDED: {
             const { groups } = action.data;
+            let ID = null;
+            for (let i = 0; i < groups.length; i += 1) {
+                if (groups[i].name === 'Generic') {
+                    ID = groups[i].id;
+                }
+            }
+            let array = [];
+            array = array.concat(ID);
             return {
                 ...state,
-                groups: groups.map((object) => { const obj = object; obj.checkbox = false; return obj; }),
+                groups: groups.map((object) => {
+                    const obj = object;
+                    if (object.name === 'Generic') { obj.checkbox = true; } else {
```

Comment:
The makes the for-loop above unneeded.  Set `array` equal to a https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set, add the ids here, and use http://stackoverflow.com/a/20070691/2336149 when setting it to `allItemID` below.